release-notes/kvm/v11.2.1.md

```diff
@@ -0,0 +1,43 @@
+## :zap: Giant Swarm Release 11.2.1 for KVM is now active for you! :zap:
+
+**With this release, horizontal pod autoscaling (HPA) is enabled by default for NGINX Ingress Controller (NGINX IC) on selected cluster profiles.**
+
+Here, you will find the NGINX IC to be more elastic thus driving down your costs. The criteria for which cluster profiles are selected will be adjusted as we collect data from running in production.
+
+Version development included work with the upstream maintainer. Prior to release, we ran trials with customers. We performed these tests on small and large clusters alike, in order to ensure stability and scalability.
+
+This version also includes an upgrade to the latest upstream NGINX IC 0.30.0, as well as improvements to other components (cluster-operator, kube-state-metrics, chart-operator) as detailed in the changelog.
```

Comment:
@sslavic for other components, are cluster-operator, kube-state-metrics, chart-operator the most important changes here?

Reply:
yes